src/App.jsx

```diff
@@ -0,0 +1,107 @@
+import './App.css';
+import {Link} from 'react-router-dom';
+import axios from 'axios';
+import { useEffect, useState } from 'react';
+
+import Header from './component/Header';
+import BestProducts from './component/BestProducts';
+import ProductList from './component/ProductList';
+import Pagination from './component/Pagination';
+import SortMenu from './component/SortMenu';
+
+function App() {
+  const [bestProducts, setBestProducts] = useState([]); 
+  const [isOpen, setOpen] = useState(false);
+  const [sortbtntext, setSortbtntext] = useState('최신순');
+  const [sortedProducts, setSortedProducts] = useState([]);
+  const pagebuttons = [1, 2, 3, 4, 5];
+  const [activePage, setActivePage] = useState(1);
+  const [sortOrder, setSortOrder] = useState('recent'); 
```

Comment:
P3:
연관있는 것들끼리 묶여 있는 것이 가독성에 더 좋습니다. 

```suggestion
  const [activePage, setActivePage] = useState(1);
  const [sortOrder, setSortOrder] = useState('recent'); 
  const pagebuttons = [1, 2, 3, 4, 5];
```